lambda-deployment-orca/lambda-deployment-orca.gradle

```diff
@@ -31,16 +31,13 @@ dependencies {
   compileOnly ("org.projectlombok:lombok:1.18.12")
   compileOnly (group: 'com.netflix.spinnaker.kork', name: 'kork-artifacts', version: "${korkVersion}")
   compileOnly (group: 'com.netflix.spinnaker.kork', name: 'kork-plugins-spring-api', version: "${korkVersion}")
-  implementation("com.amazonaws:aws-java-sdk:1.11.877")
```

Comment:
ok

Reply:
Well
```
implementation("com.netflix.awsobjectmapper:awsobjectmapper")
```
instead of
```
implementation("com.netflix.awsobjectmapper:awsobjectmapper:1.11.802")
```
but generally, yes.